### ISSUE TITLE
HEMTT Book: Search button does nothing

### ISSUE THREAD

**Issue by R3voA3** [BrettMayson/HEMTT#782]
Clicking on the search button does not show a search field and thus searching the book isn't possible.

**Comment by BrettMayson** [comment]
Yeah I don't know why this is broken, and I don't see any issues upstream
